rmw_microxrcedds_c/src/types.h

```diff
@@ -65,6 +84,11 @@ struct  rmw_context_impl_t
 #endif
   uxrSession session;
 
+#ifdef RMW_UXRCE_GRAPH
+  rmw_graph_info_t graph_info;
+#endif
+  rmw_guard_condition_t graph_guard_condition;
```

Comment:
If `rmw_node_get_graph_guard_condition` can return NULL, which I think that is possible because RCL should check the result. (TODO: Check if RCL check the output). This can be:

```suggestion
#ifdef RMW_UXRCE_GRAPH
  rmw_graph_info_t graph_info;
  rmw_guard_condition_t graph_guard_condition;
#endif
```

Reply:
It would break rcl_node_init; `get_graph_guard_condition` cannot return NULL.